share/spice/iplookup/iplookup.js

```diff
@@ -30,7 +37,7 @@
             templates: {
                 group: 'text',
                 options: {
-                    content: api_result.ddg.blacklists.length ? Spice.iplookup.content : null,
```

Comment:
P.S Please put return types for these DDG Javascript functions on the documentation! :smiley: